source/draft_api.rst

```diff
@@ -717,13 +758,15 @@ In addition, each user has a role within each project to which they belong:
 * data viewer
```

Comment:
Question: In the schema, 'data viewer' is listed as 'spectator'. Which are we going with?